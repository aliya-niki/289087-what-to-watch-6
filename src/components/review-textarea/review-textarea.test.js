import React from 'react';
import {render, screen} from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import ReviewTextarea from './review-textarea';

const mockText = `It is certainly a magical and childlike way of storytelling, even if the content is a little more adult.`;

it(`ReviewTextarea should render correctly`, () => {
  render(
      <ReviewTextarea onCommentChange={jest.fn()} disabled={false}/>
  );

  userEvent.type(screen.getByTestId(`review-textarea`), mockText);

  expect(screen.getByDisplayValue(mockText)).toBeInTheDocument();
});
