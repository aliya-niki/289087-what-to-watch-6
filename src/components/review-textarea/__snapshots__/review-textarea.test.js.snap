// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewTextarea should render correctly 1`] = `
<div>
  <textarea
    class="add-review__textarea"
    data-testid="review-textarea"
    id="review-text"
    maxlength="400"
    minlength="50"
    name="review-text"
    placeholder="Review text"
  />
</div>
`;
